jaas/pom.xml

```diff
@@ -249,7 +249,6 @@
             <groupId>joda-time</groupId>
             <artifactId>joda-time</artifactId>
             <version>2.4</version>
```

Comment:
Unfortunately, it's not working. We must specify the version. I can leave this one, or use the one from boot dependencies - 2.9.5